.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+name: Java CI
+
+on: [push, pull_request]
+#on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Java ${{ matrix.java }}
+    strategy:
+      matrix:
+        java: [8, 11]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK ${{ matrix.java }}
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+      - name: Cache Maven packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+      - name: Log github.ref
+        run: echo "${{ github.ref }}"
+      - name: Build with Maven
+        run: mvn clean install -Psecurity-checks -Pjsdoc -Dmaven.javadoc.skip=true -Djsdoc.skip.typedoc=true --batch-mode --show-version
+      - name: SonarCloud
+        if: ${{ github.ref == 'refs/heads/master' }}
+        run: mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent install sonar:sonar; fi -Dsonar.organization=primefaces -Dsonar.host.url=https://sonarcloud.io -Dsonar.login=${{secrets.SONAR_TOKEN}}
```

Comment:
I don't think the `fi` work here.  I am trying it in PFE and it complains about `fi` I would think the IF alone is enough to run this step?

Reply:
https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions
I also think the if alone should do it´s job. Not totally sure whether i compare refs github.ref correct.

Reply:
The refs look correct to me in the PFE build I can see it comparing them correctly.

Reply:
This PR in PFE shows it skipped the Sonar task:
![image](https://user-images.githubusercontent.com/4399574/95345218-82533e00-0888-11eb-8f19-1e2f07eb528f.png)